File: QViz.NET/Periscope.QViz/JSON/APIHeaderValue.cs
using System;

namespace Periscope.QViz.JSON
{
	/// <summary>
	/// JSON Model for Test API Header Value Details
	/// </summary>
	public class APIHeaderValue
	{
		/// <summary>
		/// Identifier of the API Header Value Object
		/// </summary>
		public string headerValueId;
		
		/// <summary>
		/// Key of the API HTTP Header Object
		/// </summary>
		public string key;
		
		/// <summary>
		/// Value of the API HTTP Header Object
		/// </summary>
		public string value;
		
		/// <summary>
		/// Date and Time on which this Model was Created
		/// </summary>
		public DateTime createdOn;
		
		/// <summary>
		/// Date and Time on which this Model was last Updated
		/// </summary>
		public DateTime updatedOn;

		/// <summary>
		/// Default Constructor
		/// </summary>
		public APIHeaderValue()
		{
			this.headerValueId = "";
			this.key = "";
			this.value = "";
			this.createdOn = DateTime.Now;
			this.updatedOn = DateTime.Now;
		}
	}
}

File: QViz.Java/src/main/java/com/periscope/qviz/json/APIHeaderValue.java
package com.periscope.qviz.json;

import java.util.Date;

/**
 * JSON Model for Test API Header Value Details
 */
public class APIHeaderValue {

	/**
	 * Identifier of the API Header Value Object
	 */
	public String headerValueId;

	/**
	 * Key of the API HTTP Header Object
	 */
	public String key;

	/**
	 * Value of the API HTTP Header Object
	 */
	public String value;

	/**
	 * Date and Time on which this Model was Created
	 */
	public Date createdOn;

	/**
	 * Date and Time on which this Model was last Updated
	 */
	public Date updatedOn;

	/**
	 * Default Constructor
	 */
	public APIHeaderValue() {
		this.headerValueId = "";
		this.key = "";
		this.value = "";
		this.createdOn = null;
		this.updatedOn = null;
	}

}
